public/app/features/dashboard/components/Inspector/PanelInspector.tsx

```diff
@@ -139,9 +139,8 @@ export class PanelInspector extends PureComponent<Props, State> {
             />
           </div>
         )}
-        <div style={{ border: '1px solid #666' }}>
```

Comment:
Removed this border as it was not theme agnostic and I could not really see the value of it now. If anyone has strong feelings about keeping it I don't mind adding it with correct colors.

Reply:
perfect -- that was me... hack hack hack.

I put all the pretend tab stuff there waiting for it to get replaced :)